ci/brew-install-node.nvm.inc.sh

```diff
@@ -12,26 +12,34 @@ EOF
 )"
 brew_install "${BREW_FORMULAE}"
 unset BREW_FORMULAE
+echo_done
 
-(
+[[ ! -f .nvmrc ]] || (
     set +u
     source $(brew --prefix nvm)/nvm.sh --no-use
 
-    if [[ -f .nvmrc ]]; then
-        nvm install
-    else
-        nvm install node
-    fi
-    if nvm alias system | grep -q system; then
+    nvm install
+
+    if nvm list --no-colors | grep -q system; then
         nvm reinstall-packages system
+
+        # the command above skips 'npm'
+        SYSTEM_NPM_VSN=$(nvm use system >/dev/null; npm --version)
```

Comment:
So the `nvm use system` here is purely to find out the systems npm version and the used nvm version is then overriden by line 40?

Reply:
Yup